.github/workflows/ci.yml

```diff
@@ -9,9 +9,9 @@ jobs:
       with:
         fetch-depth: 0
     - name: Setup Nodejs
-      uses: actions/setup-node@v2
+      uses: dcodeIO/setup-node-nvm@v4
       with:
-        node-version: 12
+        node-version: "${{ env.NODE_VER }}"
```

Comment:
Check out this line from the frontend-platform PR: https://github.com/edx/frontend-platform/pull/259/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR19

We need to set NODE_VER from .nvmrc like it does there.